optaplanner-benchmarks/optaplanner-perf-benchmark/src/main/java/org/jboss/qa/brms/performance/localsearch/vrp/moveselector/VRPSubChainChangeMoveSelectorBenchmark.java

```diff
@@ -13,7 +13,7 @@ public class VRPSubChainChangeMoveSelectorBenchmark extends AbstractVRPMoveSelec
 
     @Override
     public List<MoveSelectorConfig> createMoveSelectorConfigList() {
-        return MoveSelectorConfigurations.createSubChainChangeMoveSelectorList();
+        return MoveSelectorConfigurations.createSubListChangeMoveSelectorList();
```

Comment:
@dupliaka The question is, should we also rename the benchmark class to `VRPSubListChangeMoveSelectorBenchmark.java`? We may lose continuity in Horreum reports. But of course, it doesn't make sense to have a `*SubChain*Benchmark` that doesn't test subChain move selectors but subList move selectors, that would be confusing.

Reply:
Lets change it